### PR TITLE
Add Euclidean Algorithm in PowerShell

### PR DESCRIPTION
**Pull request by PaddyKe**

### CODE REVIEW

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    while ($b -ne 0) {
+        $tmp = $b
+        $b = $a % $b
+        $a = $tmp
+    }
+
+    return $a
+}
+
+Write-Host $(Mod-Euclid $(64 * 67) $(64 * 81))
```
Comment (c252) [inline contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: I would add this bit of text to keep it consistent with other implementations:

`Write-Host` $("Modulus-based euclidean algorithm result: " $(Mod-Euclid $(64 * 67) $(64 * 81)))`

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Write-Host $(Sub-Euclid $(128 * 12) $(128 * 77))
```
Comment (c252) [inline contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: Same thing here.
I would add this bit of text to keep it consistent with other implementations:
``` Write-Host $("Subtraction-based euclidean algorithm result: " $(Sub-Euclid $(64 * 67) $(64 * 81))) ```

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 [hunk 3]
```diff
@@ -0,0 +1,30 @@
+function Sub-Euclid($a, $b) {
```
Comment (c252) [inline contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: For consistency I think the function should be called ```euclid_sub```

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            $a = $a - $b
+        } else {
+            $b = $b - $a
+        }
+    }
+    
+    return $a
+}
+
+function Mod-Euclid($a, $b) {
```
Comment (c252) [inline contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: ```euclid_mod```

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 (hunk 1 above)
Comment (PaddyKe) [reply contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: I’ll add this as soon as I get back to my PC.

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 (hunk 4 above)
Comment (PaddyKe) [reply contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: I also thought about this but I wanted the methodnames to be based on the Microsoft naming convention for Cmdlets.

https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands

But I’m not fixed to this idea, so I can change it if you like the other names more.

Comment (leios) [reply contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: Yeah, use the style guidelines here.

**Comment by PaddyKe** [comment]
I added more meaningful Outputs. I think it should be clearer by now what the outputs mean.

**Comment by c252** [comment]
Hey, sorry for the late response. I would merge this PR, but I'm on Linux and don't actually have a way to verify if it works. So if one of the other devs could test that, we can merge it.

**Comment by leios** [comment]
Does anyone have a Windows device to test this with? If not, I might be able to use a VPN?

**Comment by dovisutu** [comment]
I have a Windows device (win10) but I can't use it today... Tomorrow I'll try, and if I don't respond, I have to wait until next week.

**Comment by dovisutu** [comment]
BTW, Please resolve the conflicts in `.editorconfig` and `book.json`.
This PR should be merged after resolving these and approval reviewing by @leios or other members.

**Comment by PaddyKe** [comment]
I've fixed the merge conflicts. Looks like Travis-ci had problems loading the mathjax plugin. When I run "gitbook serve" locally on my PC, it works without errors.

File: contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1 (hunk 3 above)
Comment (leios) [reply contents/euclidean_algorithm/code/powershell/euclidean_algorithm.ps1]: I think we should follow the style guidelines here, so this inconsistency is fine.

**Comment by leios** [comment]
This is a known problem. As it seems like everything is resolved, I am happy to merge this.

**Pull request merged**